airbyte-cdk/java/airbyte-cdk/airbyte-commons/src/main/java/io/airbyte/commons/logging/LoggingHelper.java

```diff
@@ -22,6 +22,7 @@ public enum Color {
     YELLOW_BACKGROUND("\u001b[43m"), // destination
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 13]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->I had no idea this existed. <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34865?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->